build/checkversion.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable camelcase */
```

Comment:
Please add the short copyright header to new files:

```
/*! @license
 * Shaka Player
 * Copyright 2016 Google LLC
 * SPDX-License-Identifier: Apache-2.0
 */
```

Also, I think it would be preferable to follow the same eslint style as the rest of the project.  No need to disable this rule and keep the same names as we would have in Python style.